src/analyze/analyze.c

```diff
@@ -2381,9 +2381,9 @@ static int parse_argv(int argc, char *argv[]) {
                 return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
                                        "Option --user is not supported for cat-config right now.");
 
-        if (arg_root && !streq_ptr(argv[optind], "cat-config"))
+        if (arg_root && !STRPTR_IN_SET(argv[optind], "cat-config", "verify"))
                 return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
-                                       "Option --root is only supported for cat-config right now.");
+                                       "Option --root is only supported for cat-config and verify right now.");
```

Comment:
actually, we generally suffix cmdline switches that take params with "=" in docs and log messages. We should do so here too. Yes, this was previously wrong too, but if you touch this line, please fix this too. i.e.:

`Option --root is only…` → `Option --root= is only…`